apps/condo/lang/en/en.json

```diff
@@ -1577,5 +1577,65 @@
   "billing.category.internet.name": "Internet",
   "billing.category.radio.name": "Radio",
   "billing.category.intercom.name": "Intercom",
-  "billing.category.telecom.name": "Telecom"
+  "billing.category.telecom.name": "Telecom",
+  "banking.integration.SBBOL.name": "SBBOL",
+  "banking.integration.1CClientBankExchange.name": "1C",
+  "banking.category.maintenance_and_current_repair_of_common_property_in_an_apartment_building.name": "Maintenance and current repair of common property in an apartment building",
+  "banking.category.apartment_building_management.name": "Apartment building management",
+  "banking.category.overhaul.name": "Overhaul",
+  "banking.category.drainage.name": "Drainage",
+  "banking.category.water_supply.name": "Water supply",
+  "banking.category.garbage_collection_and_disposal.name": "Garbage collection and disposal",
+  "banking.category.gas.name": "Gas",
+  "banking.category.hot_water_supply.name": "Hot water supply",
+  "banking.category.additional_services.name": "Additional services",
+  "banking.category.thermal_energy.name": "Thermal energy",
+  "banking.category.cold_water_supply.name": "Cold water supply",
+  "banking.category.electric_energy.name": "Electric energy",
+  "banking.costItem.emergency_service.name": "Emergency service",
+  "banking.costItem.rent.name": "Rent",
+  "banking.costItem.banking_services.name": "Banking services",
+  "banking.costItem.maintenance_and_operation.name": "Maintenance and operation",
+  "banking.costItem.overhaul.name": "Overhaul",
+  "banking.costItem.water_supply.name": "Water supply",
+  "banking.costItem.compensation_for_damage.name": "Compensation for damage",
+  "banking.costItem.garbage_collection_and_disposal.name": "Garbage collection and disposal",
+  "banking.costItem.gas.name": "Gas",
+  "banking.costItem.deratization_and_disinfestation.name": "Deratization and disinfestation",
+  "banking.costItem.materials_for_repair_and_maintenance.name": "Materials for repair and maintenance",
+  "banking.costItem.purchase_of_machinery_and_equipment.name": "Purchase of machinery and equipment",
+  "banking.costItem.employee_wages.name": "Employee wages",
```

Comment:
What exactly do you mean by "wages".
Why not "salary"?

A salary is when you pay employees the same amount each pay period regardless of how much they work. In contrast, a wage is an hourly rate you pay employees based on how much time they put in during a pay period.

Reply:
of course I'll rename it to salary. I used a translator